### DISCUSSION TITLE
Feature Request: Support for child entries

### DISCUSSION THREAD

**Discussion by daltonrooney** [trendyminds/isolate#33] [Ideas]
Me again. :-)

Another feature that I think would be super helpful would be better support for structures. I would love to be able to assign a user (or [group](#8 )) to an entry within a structure, and all child entries (whether they exist now or are created in the future). 

This might include the ability to create new child entries under the parent, assuming the user permissions allow it.

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
